src/modules/mechanism_start.json

```diff
@@ -20,22 +20,6 @@
                     "NAME": "init"
                 }
             },
-            {
```

Comment:
Why are we getting rid of the periodic method for mechanisms?   I don't think we want students to be creating a background thread and they need someway of firing events